client/blocks/reader-combined-card/index.jsx

```diff
@@ -50,13 +50,13 @@ class ReaderCombinedCardComponent extends React.Component {
 		this.recordRenderTrack();
 	}
 
-	componentWillReceiveProps( nextProps ) {
+	componentDidUpdate( prevProps ) {
 		if (
-			this.props.postKey.feedId !== nextProps.postKey.feedId ||
-			this.props.postKey.blogId !== nextProps.postKey.blogId ||
-			size( this.props.posts ) !== size( nextProps.posts )
+			this.props.postKey.feedId !== prevProps.postKey.feedId ||
+			this.props.postKey.blogId !== prevProps.postKey.blogId ||
+			size( this.props.posts ) !== size( prevProps.posts )
 		) {
-			this.recordRenderTrack( nextProps );
+			this.recordRenderTrack( this.props );
```

Comment:
can drop passing props here. and I think we could drop the default in `recordRenderTrack` too.

Reply:
Ah, good catch. Thanks!